File: calculator2.c
#include <stdio.h>

int main() {
    float num1, num2;

    printf("Masukkan angka pertama: ");
    scanf("%f", &num1);

    printf("Masukkan angka kedua: ");
    scanf("%f", &num2);

    // Menampilkan semua hasil operasi
    printf("\nHasil operasi:\n");
    printf("%.2f + %.2f = %.2f\n", num1, num2, num1 + num2);
    printf("%.2f - %.2f = %.2f\n", num1, num2, num1 - num2);
    printf("%.2f * %.2f = %.2f\n", num1, num2, num1 * num2);

    // Pengecekan pembagian dengan nol
    if (num2 != 0) {
        printf("%.2f / %.2f = %.2f\n", num1, num2, num1 / num2);} 
        else {printf("Tidak dapat membagi dengan nol.\n");}
        
    return 0;}

File: calculator2.py
def kalkulator_lengkap(angka1, angka2):

  penjumlahan = angka1 + angka2
  print(f"Penjumlahan: {angka1} + {angka2} = {penjumlahan}")


  pengurangan = angka1 - angka2
  print(f"Pengurangan: {angka1} - {angka2} = {pengurangan}")


  perkalian = angka1 * angka2
  print(f"Perkalian: {angka1} * {angka2} = {perkalian}")


  if angka2 != 0:
    pembagian = angka1 / angka2
    print(f"Pembagian: {angka1} / {angka2} = {pembagian}")
  else:
    print("Tidak dapat membagi dengan nol.")


angka_pertama = float(input("Masukkan angka pertama: "))
angka_kedua = float(input("Masukkan angka kedua: "))

kalkulator_lengkap(angka_pertama, angka_kedua)